src/__tests__/format.test.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from '@jest/globals';
+
+import { truncateAddress, formatTimestampToDate, formatDataNumber, calculateUSDGas } from '~/utils/format';
+
+describe('truncateAddress', () => {
```

Comment:
All these describe functions could be nested in one describe:

`describe('format functions', () => {}`

Just to make the hierarchy more clear, but it isn't required though.